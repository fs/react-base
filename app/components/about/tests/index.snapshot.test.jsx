import React from 'react';
import renderer from 'react-test-renderer';
import About from '../';

describe('About', () => {
  const renderComponent = () => renderer.create(<About />);

  it('renders correctly', () => {
    expect(renderComponent().toJSON()).toMatchSnapshot();
  });
});
