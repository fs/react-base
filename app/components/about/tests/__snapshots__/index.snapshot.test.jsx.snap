// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders correctly 1`] = `
<div
  className="container"
>
  <article>
    <h1>
      About
    </h1>
    <p>
      Kick-start your new web application based on React and Redux technologies.
    </p>
    <a
      onClick={[Function]}
      style={Object {}}
    >
      show details...
    </a>
  </article>
</div>
`;
